nbinteract/pull_from_remote.py

```diff
@@ -85,18 +84,18 @@ def pull_from_remote(**kwargs):
 
     # Generate repo url
     repo_url = "https://"
```

Comment:
I should have mentioned this earlier. Could you refactor the following code to something like `repo_url = _generate_repo_url(domain, account, repo_name)` so there isn't too much code in `pull_from_remote`?